ui/test/e2e/specs/test.spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable ui-testing/no-disabled-tests */
+
+describe('DAPP Offer Up E2E Test Cases', () => {
+  context('Test commands', () => {
+    it(`should complete Keplr setup by  importing an existing wallet using 24 word phrase`, () => {
+      cy.setupWallet().then((setupFinished) => {
+        expect(setupFinished).to.be.true;
+      });
+      cy.visit('/');
+    });
+    it(`should reject connection with wallet`, () => {
+      const alertShown = cy.stub().as('alertShown');
+      cy.on('window:alert', alertShown);
+
+      cy.contains('Connect Wallet').click();
```

Comment:
I'm struggling to get it to work.

<details><summary>yarn test:e2e results</summary>

```
$ SECRET_WORDS="powder bicycle deputy will move hen romance pear spring chest yard hunt cruise lumber host plug list trend alone frame sick define february copper" yarn test:e2e
yarn run v1.22.21
$ yarn workspace offer-up-ui test:e2e
$ EXTENSION=keplr synpress run --configFile=test/e2e/synpress.config.cjs
It looks like this is your first time using Cypress: 12.17.3

✔  Verified Cypress! /home/connolly/.cache/Cypress/12.17.3/Cypress

Opening Cypress...

=============================================================================

  (Run Starting)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Cypress:        12.17.3                                                                        │
  │ Browser:        Chrome 123                                                                     │
  │ Node Version:   v20.11.1 (/home/connolly/.nvm/versions/node/v20.11.1/bin/node)                 │
  │ Specs:          1 found (test.spec.js)                                                         │
  │ Searched:       test/e2e/specs/**/*spec.{js,jsx,ts,tsx}                                        │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘


────────────────────────────────────────────────────────────────────────────────────────────────────
                                                                                                    
  Running:  test.spec.js                                                                    (1 of 1)


  DAPP Offer Up E2E Test Cases
    Test commands
      ✓ should complete Keplr setup by  importing an existing wallet using 24 word phrase (28048ms)
      1) should reject connection with wallet
      2) should accept connection with wallet
      3) should reject make an offer transaction
      4) should confirm make an offer transaction


  1 passing (2m)
  4 failing

  1) DAPP Offer Up E2E Test Cases
       Test commands
         should reject connection with wallet:
     AssertionError: Timed out retrying after 30000ms: Expected to find content: 'Connect Wallet' but never did.
      at Context.eval (webpack:///./specs/test.spec.js:15:9)

  2) DAPP Offer Up E2E Test Cases
       Test commands
         should accept connection with wallet:
     AssertionError: Timed out retrying after 30000ms: Expected to find content: 'Connect Wallet' but never did.
      at Context.eval (webpack:///./specs/test.spec.js:25:9)

  3) DAPP Offer Up E2E Test Cases
       Test commands
         should reject make an offer transaction:
     AssertionError: Timed out retrying after 30000ms: Expected to find content: 'Make an Offer' but never did.
      at Context.eval (webpack:///./specs/test.spec.js:34:9)

  4) DAPP Offer Up E2E Test Cases
       Test commands
         should confirm make an offer transaction:
     AssertionError: Timed out retrying after 30000ms: Expected to find content: 'Make an Offer' but never did.
      at Context.eval (webpack:///./specs/test.spec.js:48:9)




  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        5                                                                                │
  │ Passing:      1                                                                                │
  │ Failing:      4                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  4                                                                                │
  │ Video:        true                                                                             │
  │ Duration:     2 minutes, 28 seconds                                                            │
  │ Spec Ran:     test.spec.js                                                                     │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘


  (Screenshots)

  -  /home/connolly/projects/dapp-offer-up/ui/test/e2e/screenshots/test.spec.js/DAPP       (945x848)
     Offer Up E2E Test Cases -- Test commands -- should reject connection with wallet               
      (failed).png                                                                                  
  -  /home/connolly/projects/dapp-offer-up/ui/test/e2e/screenshots/test.spec.js/DAPP       (945x848)
     Offer Up E2E Test Cases -- Test commands -- should accept connection with wallet               
      (failed).png                                                                                  
  -  /home/connolly/projects/dapp-offer-up/ui/test/e2e/screenshots/test.spec.js/DAPP       (945x848)
     Offer Up E2E Test Cases -- Test commands -- should reject make an offer transact               
     ion (failed).png                                                                               
  -  /home/connolly/projects/dapp-offer-up/ui/test/e2e/screenshots/test.spec.js/DAPP       (945x848)
     Offer Up E2E Test Cases -- Test commands -- should confirm make an offer transac               
     tion (failed).png                                                                              


  (Video)

  -  Started compressing: Compressing to 32 CRF                                                     
  -  Finished compressing: 10 seconds                                                
    Compression progress:  100%

  -  Video output: /home/connolly/projects/dapp-offer-up/ui/test/e2e/videos/test.spec.js.mp4


=============================================================================

  (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✖  test.spec.js                             02:28        5        1        4        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✖  1 of 1 failed (100%)                     02:28        5        1        4        -        -  

Cypress run results: 5 total tests, 1 passed, 4 failed
error Command failed with exit code 4.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 4
Command: /home/connolly/.nvm/versions/node/v20.11.1/bin/node
Arguments: /home/connolly/.cache/node/corepack/yarn/1.22.21/lib/cli.js test:e2e
Directory: /home/connolly/projects/dapp-offer-up/ui
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
error Command failed with exit code 4.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
</details>

Reply:
I tried again with the right mnemonic:

```
SECRET_WORDS="spike siege world rather ordinary upper napkin voice brush oppose junior route trim crush expire angry seminar anchor panther piano image pepper chest alone" yarn test:e2e
```

but the address shown in the test is `agoric1p2aqakv3ulz4qfy2nut86j9gx0dx0yw09h96md`. I don't know where that's coming from.
The "spike siege..." words correspond to `agoric1rwwley550k9mmk6uq6mm6z4udrg8kyuyvfszjk`.

Reply:
@dckc This is actually a bug in [`@agoric/synpress`](https://github.com/agoric-labs/synpress). 

At the moment, we hardcode the mnemonic:
```
'orbit bench unit task food shock brand bracket domain regular warfare company announce wheel grape trust sphere boy doctor half guard ritual three ecology',
```
Adding an issue for it in `@agoric/synpress`

Reply:
I've added it as an [over here](https://github.com/agoric-labs/synpress/issues/40).